test/api/detection_test.cpp

```diff
@@ -1,6 +1,7 @@
 #include <gtest/gtest.h>
 
 #include <seqan3/alphabet/cigar/cigar.hpp>
+// #include <seqan3/core/debug_stream.hpp>  // include for debugging
```

Comment:
delete this?

Reply:
I left this in the code, as I left some commented out debug stream lines in code. I don't know, if removing this line would be nicer.